tasks/libs/common/utils.py

```diff
@@ -301,7 +301,7 @@ def get_build_flags(
                 extldflags += ",-no_warn_duplicate_libraries "
         except ValueError:
             print(
-                "Could not determine XCode version, not adding -no_warn_duplicate_libraries to extldflags",
+                "Warning: Could not determine XCode version, not adding -no_warn_duplicate_libraries to extldflags",
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** You can wrap this into a yellow color message‏

Reply:
Right, fixed in [fc4c592](https://github.com/DataDog/datadog-agent/pull/26607/commits/fc4c592ddd374167f924e188dec5c1a6f5206e95).